swagger2.0/oic.wk.scenemember.swagger.json

```diff
@@ -0,0 +1,220 @@
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/SceneMemberResURI" : {
+      "get": {
+        "description": "Single Link that models a scene member.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/interface-baseline"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description" : "",
+            "x-example": {
+              "rt": ["oic.wk.scenemember"],
+              "id": "0685B960-FFFF-46F7-BEC0-9E6234671ADC1",
+              "n": "my binary switch (for light bulb) mappings",
+              "link": {
+                "href": "binarySwitch",
+                "rt":   ["oic.r.switch.binary"],
+                "if":   ["oic.if.a", "oic.if.baseline"],
+                "eps": [
+                  {"ep": "coap://[fe80::b1d6]:1111", "pri": 2},
+                  {"ep": "coaps://[fe80::b1d6]:1122"},
+                  {"ep": "coap+tcp://[2001:db8:a::123]:2222", "pri": 3}
+                ]
+              },
+              "SceneMappings": [
+                {
+                  "scene":           "off",
+                  "memberProperty":  "value",
+                  "memberValue":     "true"
+                },
+                {
+                  "scene":           "Reading",
+                  "memberProperty":  "value",
+                  "memberValue":     "false"
+                },
+                {
+                  "scene":           "TVWatching",
+                  "memberProperty":  "value",
+                  "memberValue":     "true"
+                }
+              ]
+            },
+            "schema": {
+              "$ref": "#/definitions/SceneMember"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-baseline" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.baseline"]
+    },
+    "interface-all" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "SceneMember": {
+      "properties": {
+        "rt": {
+          "description": "Resource Type of the Resource",
+          "items": {
+            "enum": ["oic.wk.scenemember"],
+            "type": "string"
+          },
+          "minItems": 1,
+          "readOnly": true,
+          "type": "array"
+        },
+        "SceneMappings": {
+          "description": "array of mappings per scene, can be one(1)",
+          "items": {
+            "properties": {
+              "memberProperty": {
+                "description": "property name that will be mapped",
+                "readOnly": true,
+                "type": "string"
+              },
+              "memberValue": {
+                "description": "value of the Member Property",
+                "readOnly": true,
+                "type": "string"
+              },
+              "scene": {
```

Comment:
I don't see "post" defined in this schema.  Should "scene" be listed as readOnly: true for this schema?  Not sure...

Reply:
True, but for consistency with every other schema we should be explicit about it when something is RO.  Since we're refactoring everything it feels like a good opportunity to do that now.